jetty-core/jetty-deploy/src/main/java/org/eclipse/jetty/deploy/App.java

```diff
@@ -61,6 +83,11 @@ public AppProvider getAppProvider()
         return _provider;
     }
 
+    public Map<String, String> getProperties()
```

Comment:
Should the properties be private to the `App`?

Reply:
No, there is a use-case for a different provider mutating properties after construction.